geometry_msgs/Point sourcePosition
geometry_msgs/Point targetPosition
---
bool isReachable